code/modules/admin/verbs/adminsay.dm

```diff
@@ -26,15 +26,16 @@
 	set name = "Msay"
 	set hidden = 1
 
-	if(!check_rights(R_ADMIN|R_MOD|R_MENTOR))	return
```

Comment:
Keep this as is, I meant for mentor to ONLY have msay

Reply:
so admins don't get msay?

Reply:
fixed nvm